src/container/container_iv.c

```diff
@@ -1784,8 +1784,10 @@ ds_cont_find_hdl(uuid_t po_uuid, uuid_t coh_uuid, struct ds_cont_hdl **coh_p)
 			}
 		}
 		ds_pool_child_put(pool_child);
-		D_INFO(DF_UUID ": Server handle isn't propagated yet.\n", DP_UUID(po_uuid));
-		return -DER_STALE;
+		ret = -DER_STALE;
+		DL_INFO(ret, DF_UUID ": Server handle isn't propagated yet %d.", DP_UUID(po_uuid),
```

Comment:
Minor: It's not necessary to print this known DER_STALE error code. ('ret' could be removed and print 'rc' here instead).

Reply:
I'd like to add DER_STALE to log because it is more convenient to grep from server log to know why client get that err code.